hr_recruitment_notification/data/message_subtypes.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - João Marques -->
+<odoo>
+    <data noupdate="1">
```

Comment:
It seems to me like this shouldn't be noupdate.

Reply:
Well, fine by me, it just seems that that is the standard in all Odoo message subtypes. For example, in a runbot with all core modules installed:

![image](https://user-images.githubusercontent.com/38977934/131295837-ebba3d40-a9c6-4b10-a919-478e64fa51da.png)

Reply:
Still missing.

Reply:
They are noupdate in core, and they should be that way for allowing any modification (for example, `default` flag).

Reply:
Ah that makes sense.